redis/src/main/scala/zio/redis/api/Streams.scala

```diff
@@ -123,29 +177,34 @@ trait Streams {
    * @param ids IDs of the rest of the messages
    * @return messages successfully claimed.
    */
-  final def xClaim(
-    key: String,
-    group: String,
-    consumer: String,
+  final def xClaim[SK: Schema, SG: Schema, SC: Schema, I: Schema, RK: Schema, RV: Schema](
```

Comment:
I have worries, that we have to many generic types, like here. May be we should think about some balance or that is ok and i shouldn't worry?)
@mijicd It would be cool to hear your thought about that too 😇

Reply:
It does seem to be cumbersome for the caller. 😅